v0/destinations/mailchimp/transform.js

```diff
@@ -100,6 +101,12 @@ async function responseBuilderSimple(payload, message, mailChimpConfig) {
   const basicAuth = Buffer.from(`apiKey:${mailChimpConfig.apiKey}`).toString(
     "base64"
   );
+  if (payload.status && !validStatuses.includes(payload.status)) {
+    throw new CustomError(
+      "Invalid status must be one of [subscribed, unsubscribed, cleaned, pending, transactional]",
```

Comment:
[Non-blocking comment] This is not maintainable as now the list of statuses is available in two places. We should avoid that.